src/index.spec.ts

```diff
@@ -0,0 +1,168 @@
+import 'mocha';
+import { assert } from 'chai';
+import { App, ExpressReceiver } from './index';
+import serverless from 'serverless-http';
+import {
+  delay,
+  createFakeMessageEvent,
+  createEventRequest,
+  importAppWithMockSlackClient,
```

Comment:
I think I'd try to avoid adding extra test helpers in another file where possible and stick to the current functionality - especially since anything you're adding is specific to this file. Maybe just put the smallest amount of that code you need directly in this file?  in the interest of keeping the example as simple/aligned as possible with their understanding.

Reply:
Possibly, I was trying to keep the index spec be nothing but test cases. This function was actually lifted from another test suite, I just didn't refactor it b/c that seemed like it would defeat the purpose.

I was hoping that by putting it in another file it would facilitate the need for a bolt testing suite of helpers rather than continue to keep these private to bolt.